.env.example

```diff
@@ -8,6 +8,7 @@ APP_DELETE_MINUTES=10
 APP_ENVIRONMENT_ONLY=true
 LOG_CHANNEL=daily
 APP_LOCALE=en
+APP_GOOGLE_ANALYTICS_ID=
```

Comment:
This should be referenced in the `config/pterodactyl.php` configuration file, and used in that manner in the code base, rather than direct calls to `env()`.

Doing the the way you are currently leads to issues with a cached configuration.